projects/vdk-core/src/vdk/internal/builtin_plugins/config/vdk_config.py

```diff
@@ -158,21 +156,24 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             log.info("Detected config.ini. Will try to read config.ini.")
 
             job_config = JobConfig(self.__job_path)
-            config_builder.set_value(JobConfigKeys.TEAM, job_config.get_team())
+            config_builder.set_value(JobConfigKeys.TEAM.value, job_config.get_team())
             config_builder.set_value(
-                JobConfigKeys.NOTIFIED_ON_JOB_FAILURE_USER_ERROR,
+                JobConfigKeys.TEAM, job_config.get_team()
+            )  # kept for backward compatibility
```

Comment:
Why does this need to be kept exactly?

Reply:
Because it's used in one plugin. I will update the plugin after releasing this and then remove it.